integration_tests/e2e/plan-overview.cy.ts

```diff
@@ -115,6 +115,21 @@ describe('View Plan Overview for READ_WRITE user', () => {
     cy.checkAccessibility()
   })
 
+  it('Agreed plan can show when a goal was removed', () => {
+    cy.get<{ plan: PlanType }>('@plan').then(({ plan }) => {
+      cy.addGoalToPlan(plan.uuid, DataGenerator.generateGoal({ title: 'Test Accommodation' })).then(goal => {
+        cy.addStepToGoal(goal.uuid, DataGenerator.generateStep())
+        cy.visit('/plan')
+      })
+    })
+    planOverview.agreePlan()
+    cy.get(':nth-child(3) > .govuk-link').contains('Remove').click()
```

Comment:
is there a more generic selector we can use here like `.get('a').contains` ?

Reply:
Changed it to one that's more generic